### PR TITLE
[OTA-3155] Scrolling large groups of devices leads to blank screen

### PR DESCRIPTION
**Pull request by piotrcichonmee**
Signed-off-by: Piotr Cichon <ext-piotr.cichon@here.com>

### CODE REVIEW

File: ota-plus-web/app/reactapp/src/components/devices/ContentPanel.jsx [hunk 1]
```diff
@@ -81,27 +84,77 @@ class ContentPanel extends Component {
     addNewWizard: PropTypes.func,
   };
 
+  constructor(props) {
+    super(props);
+    this.loadDevices = this.loadDevices.bind(this);
```
Comment (piotrsmol) [inline ota-plus-web/app/reactapp/src/components/devices/ContentPanel.jsx]: Redundant binding

File: ota-plus-web/app/reactapp/src/stores/DevicesStore.js [hunk 2]
```diff
@@ -188,16 +188,16 @@ export default class DevicesStore {
       });
   }
 
-  loadMoreDevices(filter = '', groupId, limit = DEVICES_LIMIT_PER_PAGE) {
+  loadMoreDevices = async (filter = '', groupId, limit = DEVICES_LIMIT_PER_PAGE, newOffset) => {
```
Comment (piotrsmol) [inline ota-plus-web/app/reactapp/src/stores/DevicesStore.js]: Remove `async`

File: ota-plus-web/app/reactapp/src/stores/DevicesStore.js [hunk 3]
```diff
@@ -188,16 +188,16 @@ export default class DevicesStore {
...
-    this.devicesCurrentPage++;
-    this.devicesOffset += limit;
-    let apiAddress = `${API_DEVICES_SEARCH}?nameContains=${filter}&limit=${limit}&offset=${this.devicesOffset}`;
+    let apiAddress = `${API_DEVICES_SEARCH}?nameContains=${filter}&limit=${limit}&offset=${newOffset}`;
     if (groupId && groupId === 'ungrouped') apiAddress += `&grouped=false`;
     else if (groupId) apiAddress += `&groupId=${groupId}`;
     return axios
       .get(apiAddress)
       .then(response => {
+        this.devicesCurrentPage++;
```
Comment (piotrsmol) [inline ota-plus-web/app/reactapp/src/stores/DevicesStore.js]: this.devicesCurrentPage += 1;

File: ota-plus-web/app/reactapp/src/components/devices/ContentPanel.jsx (hunk 1 above)
Comment (piotrcichonmee) [reply ota-plus-web/app/reactapp/src/components/devices/ContentPanel.jsx]: Done

File: ota-plus-web/app/reactapp/src/stores/DevicesStore.js (hunk 3 above)
Comment (piotrcichonmee) [reply ota-plus-web/app/reactapp/src/stores/DevicesStore.js]: Done

File: ota-plus-web/app/reactapp/src/stores/DevicesStore.js (hunk 2 above)
Comment (piotrcichonmee) [reply ota-plus-web/app/reactapp/src/stores/DevicesStore.js]: Done

**Pull request merged**